src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -109,6 +120,7 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({
           onSelect={value =>
             filterContext.setFilter("includeArtworksByFollowedArtists", value)
           }
+          my={tokens.my}
```

Comment:
Note: we need to add the padding here (e.g. like here https://github.com/artsy/force/blob/e9842c2ab8b39c32812ee3f1b344553cae63a72d/src/v2/Components/v2/ArtworkFilter/ArtworkFilters/MediumFilter.tsx#L66)